tests/code_file_manager_test.py

```diff
@@ -112,9 +121,14 @@ async def test_run_from_subdirectory(
         }
         @@code
         # Hello
-        @@end""")])
+        @@end"""
+            )
+        ]
+    )
```

Comment:
Black re-formatted a lot of strings/code in the tests as it doesn't look like we run any formatters in gh actions. Should we update github actions?

Reply:
I'm pretty sure we do run black and isort in github actions? Maybe black got updated? Or maybe we're not running black and isort on the tests directory? I'll look into this